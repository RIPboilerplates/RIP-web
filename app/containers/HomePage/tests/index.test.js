import React from 'react'
import Loadable from 'react-loadable'
import renderer from 'utils/renderIntl'

import Component from '../'

describe('NotFoundPage Component', () => {
  it('not load by default', () => {
    const component = renderer(<Component />).toJSON()
    expect(component).toBeNull()
  })

  it('does load the component', async () => {
    await Loadable.preloadAll()
    const component = renderer(<Component />).toJSON()
    expect(component).not.toBeNull()
  })
})
