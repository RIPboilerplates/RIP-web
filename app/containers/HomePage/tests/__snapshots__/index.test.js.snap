// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage Component Renders default 1`] = `
<div
  className="styles__Container-eZxbLb fCoiuO"
>
  <h1>
    <span>
      This is HomePage component!
    </span>
  </h1>
</div>
`;
